bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/OpenWebNetBindingConstants.java

```diff
@@ -173,6 +173,7 @@ public class OpenWebNetBindingConstants {
     public static final String CHANNEL_ALARM_SYSTEM_BATTERY = "battery";
     public static final String CHANNEL_ALARM_ZONE_STATE = "state";
     public static final String CHANNEL_ALARM_ZONE_ALARM = "alarm";
+    public static final String CHANNEL_ALARM_ZONE_TIMESTAMP = "timestamp";
```

Comment:
suggest to call it CHANNEL_ALARM_ZONE_ALARM_TIMESTAMP

Reply:
done